operator/validator/event_handler.go

```diff
@@ -295,12 +296,12 @@ func (c *controller) handleValidatorRemovedEvent(
 	}
 
 	// remove decided messages
-	messageID := spectypes.NewMsgID(types.GetDefaultDomain(), share.ValidatorPubKey, spectypes.BNRoleAttester)
-	store := c.ibftStorageMap.Get(messageID.GetRoleType())
-	if store != nil {
-		if err := store.CleanAllInstances(logger, messageID[:]); err != nil { // TODO need to delete for multi duty as well
-			return nil, errors.Wrap(err, "could not clean all decided messages")
-		}
+	err := c.ibftStorageMap.Each(func(role spectypes.BeaconRole, store qbftstorage.QBFTStore) error {
+		messageID := spectypes.NewMsgID(types.GetDefaultDomain(), share.ValidatorPubKey, role)
```

Comment:
We use `types.GetDefaultDomain()` as a singleton in many places right now, but I think we shouldn't do that for new code. Let's pass domain from network config. I implemented this in my PR, so you can copy changes to yours: https://github.com/bloxapp/ssv/pull/1020/commits/1429e89f6e675a75577e3f2cecfaf6e9e541b032